sidebars.js

```diff
@@ -31,8 +31,31 @@ const sidebars = {
   ourSidebar: [
     'cellular-iot',
     'quickstart',
-    'service-stack-overview',
-    'services',
+    {
+      type: 'category',
+      label: 'Services',
+      link: { 
+        type: 'generated-index', 
+        title: 'Service Stack Overview',
```

Comment:
This should be sentence case as well, plus it would be useful to add the description that was previously available on this page (feel free to tweak)

```suggestion
        title: 'Service stack overview',
        description: 'emnify offers the following high-level products and services:',
```